content/course-specific-material/introR-SAWSC-Feb17.Rmd

```diff
@@ -0,0 +1,71 @@
+---
+title: "SAWSC - North Carolina"
+date: "2017-02-07"
+slug: "introR-SAWSC-Feb17"
+output: USGSmarkdowntemplates::hugoTraining_course
+parent: Course Specific Material
+---
+February 7th - 9th in Raleigh, NC
+
+### Installation
+
+See [Before the Workshop](/intro-curriculum/Before) for information on what software should be installed prior to the course.
+
+### Tentative schedule
+
+**Day 1**
+
+-   08:30 am - 09:00 am -- Instructors available for questions
+-   09:00 am - 10:15 am -- [Introduction](/intro-curriculum/Introduction)
```

Comment:
we will actually get started at 8:30, so instructors are available for questions 8-8:30. You can follow the Arkansas training pretty closely: https://owi.usgs.gov/R/training-curriculum/course-specific-material/introR-LMGWSCAR-Oct16/. There are a few sections that we don't do or we have yet to decide. 

Day 2 
01:00 pm - 02:30 pm – Visualize: ggplot2
02:45 pm - 04:00 pm – Visualize: swmrQW for censored data and publication-quality graphs

Both of these will be "Visualize: base R"

Day 3 
10:00 am - 10:30 am – The importance of reproducibility

We won't have this. Instead, the practice time would start at 10:00